Camera.hpp

```diff
@@ -17,6 +17,7 @@ namespace spic {
              * @param backgroundColor The background color of the horizon in the camera.
              * @param aspectWidth The aspect width of the camera.
              * @param aspectHeight The aspect width of the camera.
+             * @sharedapi
```

Comment:
Bij Camera lijken ook nog getters/setters te missen voor alle properties

Reply:
Aangezien we deze niet gaan gebruiken lijkt het mij niet nodig om deze toe te voegen

Reply:
Die setters en getters heb ik in een losse pr gedaan want deze is al groot genoeg https://github.com/42IN7SPICc/spic-prj-api-cd/pull/44

En de camera gaan wij wel gebruiken